day2.md

```diff
@@ -0,0 +1,171 @@
+## Day2 Tues. 09/06/16 [Tokyo 東京]
+---
+
+10:00 - 11:00 [Tsukiji Fish Market](#fish_market)<br />
+12:00 - 14:20 [Tokyo Skytree Town](#skytree)<br />
+15:00 - 16:00 [Sensō-ji](#temple)<br />
+17:00 - 19:00 [Shinjuku & Fuunji Ramen(風雲児)](#shinjuku)<br />
+20:00         [Department Store](#store)
+
+---
+There was no better way to start our second day in Japan than going to <a name="fish_market">the Tsukiji fish market</a>!
+> "Tsukiji Market (築地市場, Tsukiji Shijō) is a large wholesale market for fish, fruits and vegetables in central Tokyo. It is the most famous of over ten wholesale markets that handle the distribution of fish, meat, produce and flowers in Tokyo. Tsukiji Market is best known as one of the world's largest fish markets, handling over 2,000 tons of marine products per day." -[Japen Guide](http://www.japan-guide.com/e/e3021.html)
+
+Many tourists will come here by 5 a.m. for the [live tuna auctions](https://www.youtube.com/watch?v=uv5DloVT3iU). Knowing sleep is the second most important thing, after food, in our lives, we saved ourselves the troubles of thinking wishfully, setting a 4 am alarm, turning it off as soon as it rings, and going right back to sleep. Instead, after a great night of sleep, we woke up naturally around 9 am and headed straight to the Tsukiji fish market with only one clear goal in mind: food!
```

Comment:
"saved ourselves the **trouble**" is a more common phrasing, but I could see "troubles" too.

I also think this is hilarious and so like you.